packages/icon/package.py

```diff
@@ -96,12 +96,17 @@ class Icon(Package):
             description='Build with hammoz and atm_phy_echam enabled.')
     variant('art', default=False, description='Build with art enabled')
     variant('ocean', default=True, description='Build with ocean enabled')
+    variant('dace', default=False, description='Build with DACE enabled')
+    variant('rttov',
+            default=False,
+            description='Build DACE with RTTOV enabled')
```

Comment:
Shouldn't this read `'Build with RTTOV enabled'`?

Reply:
fixed